crates/ark/src/modules/positron/package.R

```diff
@@ -21,6 +21,32 @@
 #' @export
 .ps.rpc.is_installed <- .ps.is_installed
 
+# Returns a list containing:
+#   * the version string if the package is installed and NULL otherwise
+#   * a minimum version to check for, with '0.0.0' as a dummy default
+#   * a logical indicating if package is installed at or above the minimum version
+#  This may seem weird, but it's impractical for positron-r to do version
+#  comparisons.
+#' @export
+.ps.rpc.packageVersion <- function(pkg, checkAgainst = "0.0.0") {
```

Comment:
```suggestion
.ps.rpc.packageVersion <- function(pkg, minimumVersion = "0.0.0") {
```

I'd be tempted to call it this to be super clear that its:
- a version
- the minimum one to check against

I'd also be tempted to default it to `NULL`, implying "no minimum version check is done" - see also `rlang::is_installed()`